modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java

```diff
@@ -505,66 +432,33 @@ public void reAssignSlotsWhenMemberLeaves(String nodeId) {
-            long endMsgId = emptySlot.getEndMessageId();
-            long slotDeleteSafeZone = getSlotDeleteSafeZone();
-            if(log.isDebugEnabled()) {
-                log.debug("Trying to delete slot. safeZone= " + getSlotDeleteSafeZone() + " startMsgID= "
-                        + startMsgId);
-            }
-            if (slotDeleteSafeZone > endMsgId) {
-                String lockKey = nodeId + SlotManagerClusterMode.class;
-                synchronized (lockKey.intern()) {
-                    HashMap<String, TreeSet<Slot>> queueToSlotMap = null;
-                    HashmapStringTreeSetWrapper wrapper = slotAssignmentMap.get(nodeId);
-                    if (null != wrapper) {
-                        queueToSlotMap = wrapper.getStringListHashMap();
-                    }
-                    if (queueToSlotMap != null) {
-                        TreeSet<Slot> currentSlotList = queueToSlotMap.get
-                                (queueName);
-                        if (currentSlotList != null) {
-                            // com.google.gson.Gson gson = new GsonBuilder().create();
-                            //get the actual reference of the slot to be removed
-                            Slot slotInAssignmentMap = null; //currentSlotList.ceiling(emptySlot);
-
-                            for (Slot slot : currentSlotList) {
-                                if (slot.getStartMessageId() == emptySlot.getStartMessageId()) {
-                                    slotInAssignmentMap = slot;
-                                }
-                            }
-
-                            if (null != slotInAssignmentMap) {
-                                if (slotInAssignmentMap.addState(SlotState.DELETED)) {
-                                    currentSlotList.remove(slotInAssignmentMap);
-                                    queueToSlotMap.put(queueName, currentSlotList);
-                                    wrapper.setStringListHashMap(queueToSlotMap);
-                                    slotAssignmentMap.set(nodeId, wrapper);
-                                    if(log.isDebugEnabled()) {
-                                        log.debug("Deleted slot from Slot Assignment Map : Slot= " +
-                                                slotInAssignmentMap);
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-                return true;
-            } else {
-                if(log.isDebugEnabled()) {
-                    log.debug("Cannot delete slot as it is within safe zone startMsgID= " + startMsgId +
-                            " safeZone= " + slotDeleteSafeZone + " endMsgId= " + endMsgId + " slotToDelete= " + emptySlot);
+        /**
+         * slot id of empty slot is empty
+         */
+        long startMsgId = emptySlot.getStartMessageId();
```

Comment:
Shall we use `//` instead of JDoc style comments here. This comment is within a method and no need to use JDoc comment for this